roles/pmm/defaults/main.yaml

```diff
@@ -22,9 +22,28 @@ pmm_agent_setup_flags:
   server-password: '{{ pmm_agent_token }}'
   server-insecure-tls: '{{ pmm_server_insecure }}'
 
-pmm_ms_folder: 'ManagedService'
-pmm_alertmanager_contactpoint_uid: managedservice_contact_point
+pmm_template_folder: '{{ pmm_default_template_folder | default("DefaultFolder")}}'
+pmm_alertmanager_contactpoint_uid: '{{ pmm_alertamanger_default_contactpoint_uid | default("DefaultContact_")}}'
+pmm_templates_prefix: '{{ pmm_default_templates_prefix | default("DefaultPrefix_")}}'
+pmm_alerting_runbook_path: 'https://localhost/runbooks/'              # path used in alert labels
+pmm_alertmanager_rule_periods:                                                                          # Used to set the "for" duration in rules
+  interval_unit: m
+  low_interval: 1
+  med_interval: 5
+  high_interval: 10
+  max_interval: 15
+pmm_alert_filters: []            # example: '{"VmalertAlertsError":{"filters":[{"label":"env","regexp":"staging","type":"MATCH"},{"label":"node_name","regexp":"db10","type":"MISMATCH"}]}}'
```

Comment:
@theTibi you have the variable defined as a list and then show a string. See [`pmm_manage_users`](https://github.com/cezmunsta-org/gascan-ansible/blob/b2c78bc258323497997074a3ed3c93b554d80e69/roles/pmm/defaults/main.yaml#L76) as an example